packages/browser-destinations/package.json

```
@@ -84,7 +84,7 @@
   "size-limit": [
     {
       "path": "dist/web/*/*.js",
-      "limit": "169 KB"
+      "limit": "188 KB"
```

```suggestion
      "limit": "188 KB"
```

I see that you bumped the size by 19 KBs. I was unable to see the previous CI error, but this should be only increased enough to pass the checks by a small margin. Do you remember if 188 was within this small margin?

The size limit isn’t meant to be a hard limit - it is just in place to alert us in case the size increases by an unexpected amount. You should totally feel fine increasing the size limit as needed (and adding a buffer of around a kilobyte).

---

I bumped it up a couple of KB over size shown in warning message